crud/common/utils.lua

```diff
@@ -199,6 +199,60 @@ function utils.tarantool_supports_uuids()
     return enabled_tarantool_features.uuids
 end
 
+local function map_to_table(map)
+    local tab = {}
+    for _, value in pairs(map) do
+        table.insert(tab, value)
+    end
```

Comment:
Here you use an assumption that pairs will return you values ordered by indexes. I'm not sure that it's correct.

Reply:
Maybe we don't need to use "recursive" approach here. It's really not obvious why `operations[id - 1] = {'=', id - 1, box.NULL}` can't rewrite some user operation.